Source/WebKit/Platform/mac/MenuUtilities.mm

```diff
@@ -160,10 +160,23 @@ - (BOOL)revealContext:(RVPresentingContext *)context shouldUseDefaultHighlightFo
 
 #if ENABLE(CONTEXT_MENU_IMAGES_FOR_INTERNAL_CLIENTS)
 
-static NSString *symbolNameForAction(const WebCore::ContextMenuAction action, bool useAlternateImage)
+enum class SymbolType : uint8_t {
+    None,
```

Comment:
How about using `std::optional` for the return type instead?

Reply:
Agreed, since you don't care about the symbol name for `SymbolType::None`, just return a `std::optional<SymbolNameWithType>`

Reply:
and `enum class SymbolType : uint8_t { Public, Private };`